build-config/resources/Ballerina.toml

```
@@ -0,0 +1,12 @@
+[package]
+org = "nuvindu_dias"
+name = "pipe"
+version = "@toml.version@"
+authors = ["Ballerina"]
+keywords = ["pipe"]
+repository = "https://github.com/Nuvindu/module-pipe"
+license = ["Apache-2.0"]
+distribution = "2201.0.1"
+
+[[platform.java11.dependency]]
+path = "../native/build/libs/pipe-native-0.1.0-SNAPSHOT.jar"
```

This should contain the placeholder to replace the version.

```suggestion
path = "../native/build/libs/pipe-native-@project.version@.jar"

```